README.md

```diff
@@ -29,7 +29,7 @@ The `sparkles-component` API supports most of the `@glimmer/component` API, incl
   * Support for setting up dependent keys to cause a properties getter to be invoked again
   * Support for tracking class fields
 * Decorator Support
-  * Support for consuming with Babel 6 (just install `@ember-decorators/babel-transform`)
+  * Support for consuming with Babel 6 (just run `ember install -S @ember-decorators/babel-transforms@^2.0.0`)
```

Comment:
Would you mind removing the word `just` here? 

I know I probably wrote it, but I really dislike using words like "just" / "simply" / etc in documentation because they are pretty demeaning...